processor/src/main/java/com/linecorp/decaton/processor/runtime/ProcessorsBuilder.java

```diff
@@ -130,7 +130,7 @@ public ProcessorsBuilder<T> thenProcess(DecatonProcessor<T> processor) {
         return thenProcess(new DecatonProcessorSupplierImpl<>(() -> processor, ProcessorScope.PROVIDED));
     }
 
-    public Processors<T> build(DecatonProcessorSupplier<byte[]> retryProcessorSupplier) {
+    Processors<T> build(DecatonProcessorSupplier<byte[]> retryProcessorSupplier) {
```

Comment:
while I'm ok with this, it is a breaking change

Reply:
https://github.com/line/decaton/blob/master/VERSIONING.md#public-apis

> A constructor/method/field that depends on at least one of internal items in its signature. e.g, com.linecorp.decaton.LoggingContext's constructor that takes com.linecorp.decaton.runtime.internal.TaskRequest as an argument.

Good point :D